backend/small_eod/files/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from letter.models import Letter
+# Create your models here.
+
+
+class File(models.Model):
+    path = models.FilePathField()
+    name = models.CharField(max_length=100)
+    letter = models.ForeignKey(to=Letter, on_delete=models.CASCADE)
+
+
+class FileSigner(models.Model):
```

Comment:
Ten model nie jest potrzebny w ogóle. Można go usunąć.

Reply:
FileSigner, tak? Bo zwykle File Ci zostawic jako template, czy bedziesz sobie sam to implementowac?